draft-ietf-emu-aka-pfs-latest.xml

```diff
@@ -1726,7 +1725,7 @@ EMSK     = MK_ECDHE[768..1279]
 
    <t>Best practice security today is to mandate forward secrecy (as
    is done in WPA3, EAP-TLS 1.3, EAP-TTLS 1.3, IKEv2, SSH, QUIC, WireGuard, Signal, etc.). It is RECOMMENDED
-   to long term completely phase out AKA without forward secrecy.</t>
+   that AKA without forward secrecy be phased out.</t>

```

Comment:
```suggestion
   that AKA without forward secrecy be phased out in the long term.</t>

```

Reply:
@jariarkko would this work?

Reply:
I am fine with any of three suggestions.

Reply:
Let's change to EAP-AKA methods without without forward secrecy